packages/frontend/src/features/tableCalculation/components/TableCalculationModal.tsx

```diff
@@ -177,7 +178,13 @@ const TableCalculationModal: FC<Props> = ({
                             <Button variant="outline" onClick={onClose}>
                                 Cancel
                             </Button>
-                            <Button type="submit"> Save </Button>
+                            <Button
+                                type="submit"
+                                data-testid="table-calculation-save-button"
+                            >
+                                {' '}
+                                Save{' '}
```

Comment:
can remove these extra spaces

Reply:
For sure! These must've been from auto formatting? 🤔